.gitignore

```diff
@@ -15,3 +15,6 @@
 # vendor/
 #
 main
+
+.idea/
+cloud-native-go-helloworld-service
```

Comment:
IMO it's handy to have excluded the binary name, sometimes I'll do a `go build` just to see if there are any errors